github-dark-element-borders.user.css

```
@@ -9,13 +9,20 @@
 @updateURL    https://raw.githubusercontent.com/StylishThemes/Feature-Override-Styles/master/github-dark-element-borders.user.css
 @license      CC-BY-SA-4.0
 
+@var checkbox enableFileTableBorders "Enable borders in file table" 0
 @var checkbox enableBorders "Enable borders" 1
 
 @preprocessor stylus
 ==/UserStyle== */
 @-moz-document regexp("^https?://((gist|guides|help|lab|launch-editor|raw|resources|status|developer)\\.)?github\\.com/((?!generated_pages/preview).)*$"),
                domain("githubusercontent.com"), domain("graphql-explorer.githubapp.com"),
                domain("www.githubstatus.com") {
+  if (enableFileTableBorders) {
+    .Details .Box-row.border-top-0:nth-child(n + 3) {
```

Prefix selectors with html body, so 

```suggestion
      html body .Details .Box-row.border-top-0:nth-child(n + 3)
```

---

You removed the opening `{`.